keps/sig-api-machinery/20190425-crd-conversion-webhook.md

```diff
@@ -736,7 +736,11 @@ Consider including folks that also work outside the SIG or subproject.
 
 ## Graduation Criteria
 
+v1beta1:
```

Comment:
could you also migrate versioning/conversion test plan here from the CRD GA KEP?

Reply:
Added.